python/ray/data/_internal/logging.yaml

```diff
@@ -25,3 +25,7 @@ loggers:
         level: DEBUG
         handlers: [file, console]
         propagate: False
+    ray.air.util.tensor_extensions:
```

Comment:
any downside of including the whole `ray.air`?

Reply:
Hmm...haven't dug through all of the `air` code, but would want to avoid writing non-Data logs to `ray-data.log` (e.g., if there's some Train stuff)